graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/GraphQLConfigurationProperties.kt

```diff
@@ -48,3 +49,12 @@ class SubscriptionConfigurationProperties {
     /** Keep the websocket alive and send a message to the client every interval in ms. Default to not sending messages */
     var keepAliveInterval: Long? = null
 }
+
+/**
+ * Play ground configuration properties.
+ */
+class PlayGroundConfigurationProperties {
+    /** playground service endpoint, defaults to '/' */
```

Comment:
this comment should be on `endpoint` I believe. Let's make it explicit -> `Prisma Playground GraphQL IDE endpoint, defaults to "/"`